### DISCUSSION TITLE
react-native-video audiotrack selection in android in fullscreen mode like in ios

### DISCUSSION THREAD

**Discussion by TheNofal** [TheWidlarzGroup/react-native-video#4291] [Q&A]
### What are you trying to do?

I want to use audio track selection on Android in fullscreen mode, like it works in iOS.

Does anyone know if this feature is available in the React Native Video player or not in Android?

### Platforms

Android

**Comment by freeboub** [comment]
It is not currently supported, but if you want to develop it, it is welcome !
